pkspace/utils/loaders.py

```diff
@@ -0,0 +1,105 @@
+import glob
+import json
+import os
+import xml.etree.ElementTree
+import cv2
+import numpy as np
+
+
+class Loader:
+    def load(self, path, split):
+        pass
+
+
+class PKspaceLoader(Loader):
+    def __get_spaces__(self, src, desc):
+        finsize = (80, 80)
+        spaces = []
+        answers = []
+        for space in desc.get('spots'):
+            points = space.get('points')
+            angle = space.get('rotation')
+            nove_pole = np.hstack(
```

Comment:
@rstevanak please let's either use English variable names, or some other language, but let us also make sure we do not mingle two of them together.